dev/com.ibm.ws.com.lightbend.microprofile.reactive.streams.zerodep/.classpath

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="bin_test" path="test/src"/>
+	<classpathentry kind="src" path="zerodep/src/main/java"/>
+	<classpathentry kind="src" path="zerodep/src/test/java"/>
+	<classpathentry exported="true" kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="lib" path="/com.ibm.websphere.org.reactivestreams.reactive-streams.1.0.2/build/libs/com.ibm.websphere.org.reactivestreams.reactive-streams.1.0.2.jar"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0"/>
```

Comment:
so before delivery you'll want to remove most of this stuff ... all of the dependency management (including unit tests) is done by bnd (in the bnd.bnd file) so the only external classpath entry you need should be for `aQute.bnd.classpath.container`